### PR TITLE
Sentry.setTags & scope.setTags

### PR DESCRIPTION
**Pull request by ryfieldsentry**
Hey, 

Docs team update - it's to add clarification on Sentry.setTags vs scope.setTags.

Here's the asana task for context: 
https://app.asana.com/0/1144049194416663/1146296934584163

Thanks

### CODE REVIEW

**Comment by dashed** [comment]
@ryfieldsentry can the screenshot images be renamed from the default autogenerated names (assuming from macOS) to something more meaningful?

**Comment by ryfieldsentry** [comment]
Hey Alberto,

Sure - I will update them.

Thanks.

**Comment by dashed** [comment]
Thanks for updating them 👍

File: src/collections/_documentation/enriching-error-data/scopes.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ nitty gritty details.  On platforms such as .NET or on Python 3.7 and later we w
 use "ambient data" to have either the hub flow with your code or the hub is already
 a singleton that internally manages the scope.
 
-Effectively this means that when you spawn a task in .NET and the execution flow is 
+Effectively this means that when you spawn a task in .NET and the execution flow is
 not supressed all the context you have bound to the scope in Sentry will flow along.
```
Comment (MimiDumpling) [inline src/collections/_documentation/enriching-error-data/scopes.md]:
```suggestion
not suppressed, all the context you have bound to the scope in Sentry will flow along.
```

File: src/collections/_documentation/enriching-error-data/scopes.md [hunk 2]
```diff
@@ -40,7 +40,7 @@ nitty gritty details.  On platforms such as .NET or on Python 3.7 and later we w
...
 If however you suppress the flow, you get new scope data.
```
Comment (MimiDumpling) [inline src/collections/_documentation/enriching-error-data/scopes.md]:
```suggestion
However, if you suppress the flow, you get new scope data.
```

File: src/collections/_documentation/enriching-error-data/scopes.md [hunk 3]
```diff
@@ -66,18 +66,34 @@ To learn what useful information can be associated with scopes see
...
-typically called `with-scope` or `push-scope` which is also very helpful 
+typically called `with-scope` or `push-scope` which is also very helpful
 if you only want to send data with one specific event.  In the following example we are using
 that function to attach a `level` and a `tag` to only one specific error:
 
 {% include components/platform_content.html content_dir='with-scope' %}
 
-While this example looks similar to `configure-scope` it's very different, in the sense that 
-`configure-scope` actually changes the current active scope, all successive calls to `configure-scope` 
+While this example looks similar to `configure-scope` it's very different, in the sense that
```
Comment (MimiDumpling) [inline src/collections/_documentation/enriching-error-data/scopes.md]:
```suggestion
While this example looks similar to `configure-scope`, it's very different, in the sense that
```

File: src/collections/_documentation/enriching-error-data/scopes.md [hunk 4]
```diff
@@ -66,18 +66,34 @@ To learn what useful information can be associated with scopes see
...
+`configure-scope` actually changes the current active scope, all successive calls to `configure-scope`
```
Comment (MimiDumpling) [inline src/collections/_documentation/enriching-error-data/scopes.md]:
```suggestion
`configure-scope` actually changes the currently active scope, all successive calls to `configure-scope`
```

**Comment by ryfieldsentry** [comment]
Thank you Mimi! I will make those changes today.

**Comment by MimiDumpling** [comment]
@ryfieldsentry Looks like some of your tests are failing. Let me know if you need help on those. 😉

**Pull request merged**